### PR TITLE
BD-2575: Hidden doc for updated email HTML editor

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2575**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_hidden/private_betas/email_html_react.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+nav_title: Email HTML Editor
+article_title: Email HTML Editor
+permalink: "/email_html_editor/"
+hidden: true
+description: "This article provides an overview of the updated HTML editor for email."
+---
+
+# Updated HTML editor for email
```
Comment (colleenwoodford) [inline _docs/_hidden/private_betas/email_html_react.md:9]: @lydia-xie We also want customers to know if they want the Link Management tab, they need to talk to their CSM or account manager to get Link Aliasing enabled. We could link to the docs for that, if possible? Let me know if you think this is a bad idea to put in the docs. This is the feature difference we've had the most questions about.

File: _docs/_hidden/private_betas/email_html_react.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+![][1]
+
+{% alert important %}
+While you can still revert to the old HTML editor by clicking **Switch to old HTML editor**, note that the old HTML editor will be deprecated in January 2025.
+{% endalert %}
+
+## How to access the updated editor
+
+To access the updated HTML email editor and link management, you must have link aliasing turned on. To turn on link aliasing, contact your account manager.
```
Comment (djhanggi) [inline _docs/_hidden/private_betas/email_html_react.md:23]:
```suggestion
To access the link management tab in the updated HTML email editor, you must have link aliasing turned on. To turn on link aliasing, contact your account manager.
```

Comment (djhanggi) [reply _docs/_hidden/private_betas/email_html_react.md:23]: WDYT?

File: _docs/_hidden/private_betas/email_html_react.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# Updated HTML editor for email
+
+> The updated HTML editor for email features a series of UI improvements, including a better code editing experience. 
+
+With the updated HTML editor, you can seamlessly switch among editors (HTML, classic, plaintext, and AMP) with the top editor navigation. These options were previously hidden in a dropdown in the old HTML editor. This updated HTML editor also creates a consistent experience across our editors in other messaging channels, allowing Braze to more efficiently build universal improvements to these editors in the future.
+
+![][1]
+
+{% alert important %}
+While you can still revert to the old HTML editor by clicking **Switch to old HTML editor**, note that the old HTML editor will be deprecated in January 2025.
```
Comment (djhanggi) [inline _docs/_hidden/private_betas/email_html_react.md:18]:
```suggestion
While you can still revert to the old HTML editor by clicking **Switch to old HTML editor**, note that the old HTML editor will be removed in January 2024.
```

File: _docs/_hidden/private_betas/email_html_react.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+![][1]
+
+{% alert important %}
+While you can still revert to the old HTML editor by clicking **Switch to old HTML editor**, note that the old HTML editor will be removed in January 2024.
+{% endalert %}
+
+## How to access the updated editor
+
+To access the **Link Management** tab in the updated HTML email editor, you must have [link aliasing]({{site.baseurl}}/user_guide/message_building_by_channel/email/templates/link_aliasing/) turned on. To turn on link aliasing, contact your account manager.
```
Comment (colleenwoodford) [inline _docs/_hidden/private_betas/email_html_react.md:23]: Can we put this in an alert or something, perhaps above the fold, so it's more clear? I'm dumb and missed it even after requesting it myself. 

Perhaps have the how to access / docs / and then the deprecation date together at the end?

Comment (lydia-xie) [reply _docs/_hidden/private_betas/email_html_react.md]: I removed the header since the how to access bit is wrapped up an alert, lmk your thoughts!

**Review by djhanggi — APPROVED** [review]

File: _docs/_hidden/private_betas/email_html_react.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+permalink: "/email_html_editor/"
+hidden: true
+description: "This article provides an overview of the updated HTML editor for email."
+---
+
+# Updated HTML editor for email
+
+> The updated HTML editor for email features a series of UI improvements, including a better code editing experience. 
+
+With the updated HTML editor, you can seamlessly switch among editors (HTML, classic, plaintext, and AMP) with the top editor navigation. These options were previously hidden in a dropdown in the old HTML editor. This updated HTML editor also creates a consistent experience across our editors in other messaging channels, allowing Braze to more efficiently build universal improvements to these editors in the future.
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/email_html_react.md:13]:
```suggestion
With the updated HTML editor, you can seamlessly switch between editors (HTML, classic, plaintext, and AMP) with the top editor navigation. These options were previously hidden in a dropdown in the old HTML editor. This updated HTML editor also creates a consistent experience across our editors in other messaging channels, allowing Braze to more efficiently build universal improvements to these editors in the future.
```

File: _docs/_hidden/private_betas/email_html_react.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+![][1]
+
+{% alert important %}
+To access the **Link Management** tab in the updated HTML email editor, you must have [link aliasing]({{site.baseurl}}/user_guide/message_building_by_channel/email/templates/link_aliasing/) turned on. To turn on link aliasing, contact your account manager.
+{% endalert %}
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/email_html_react.md:19]: This alert seems kind of randomly inserted. Maybe it should be part of an FAQ on this page? Like we could have an entry for "Where is the Link Management tab?" and "When will the old editor go away?" and whatever else may be relevant for them to know 👀

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**